src/api/db/migrate/20231204093136_create_appeals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAppeals < ActiveRecord::Migration[7.0]
+  def change
+    create_table :appeals, id: :bigint do |t|
+      t.text :reason, null: false
+      t.integer :appellant_id, null: false, foreign_key: { to_table: :users }
+      t.integer :decision_id, null: false, foreign_key: true
```

Comment:
This should be a `bigint` since this is what the decisions table's primary key is using, see https://github.com/openSUSE/open-build-service/blob/9b50289ba1a59378493f6209e9218b25c7f6c28c/src/api/app/models/decision.rb#L44-L46